docs/agent/secrets.md

```diff
@@ -226,24 +253,47 @@ with 2 fields:
 
 Example:
 
-Here is a dummy script prefixing every secret with `decrypted_`:
-
-```py
-#!/usr/bin/python
+Here is a dummy Go program prefixing every secret with `decrypted_`:
```

Comment:
why replace the python script with Go? Is it only because it's easier to generate a binary from Go code than python?

Asking because I assume most users are more familiar with python than Go, especially since custom checks are in python.

Reply:
It come from the Windows branch. You can't use a simple python script on Windows since it's not a Win32 application (ie: you can't execute it).

So we update the demo script to use Go so users can easily generate a static binary to test the secret feature.

Reply:
ok, works for me for these docs. We may want to still provide a python example in the future (and provide a note about Windows, we could suggest Windows + python users to generate windows executable using 3rd-party libs such as `py2exe` or `PyInstaller`)